core/src/main/java/com/novoda/noplayer/mediaplayer/CheckBufferHeartbeatCallback.java

```diff
@@ -4,9 +4,7 @@
 import com.novoda.noplayer.Player;
 import com.novoda.noplayer.VideoPosition;
 
-@SuppressWarnings("PMD.RedundantFieldInitializer")
-        // we're being very explicit with our default field values, not a bad thing!
-class CheckBufferHeartbeatCallback implements Heart.Heartbeat.Callback<Player> {
+public class CheckBufferHeartbeatCallback implements Heart.Heartbeat.Callback<Player> {
```

Comment:
The only reason I was seen for this class being public is the fact that we want to access the buffer listener, but given the class is implementing an interface I thought perhaps the buffer listener can be part of that interface it is implemented,

Then we can make this be package protected as it was before